app/components/AllOrders.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+export class AllOrders extends React.Component {
```

Comment:
if we're going to have this be a `class` object, don't we need a `constructor` items the class, e.g.,:
`constructor(props) {
  super(props)
}`

without this `constructor` item, I don't know how the props will be passed to the `render` method.